lib/src/data/sports.dart

```diff
@@ -164,15 +163,19 @@ class SportsGame {
 	/// won, and which score to get depending on [isHome].
 	final Scores? scores;
 
+	/// The URL to the livestream for this game
+	late String? livestreamUrl;
```

Comment:
```suggestion
	final String? livestreamUrl;
```
1. This shouldn't be `late`. `late` is for when the value is unknown when class is created, but `null` isn't appropriate to represent that value. Then, the real value is set then it's never "unset" again. You have the right idea here: create a game with an unknown link, then days or weeks later add the link, and never change it. However, `late` is much shorter-term, more like a few lines of code later than actual days. Besides...
2. This should be final. As you can see in the Sports data model, updating a game consists of making a copy with the relevant value replaced, and uploading that _new_ game to the database. This is different from other paradigms where you'd do `game.link = newLink`. We do it this way because the admin can change pretty much everything about the game all at once in the UI, so it makes sense to just create a new game with the relevant properties.